.buildkite/ftr_configs.yml

```diff
@@ -48,6 +48,8 @@ disabled:
   - x-pack/test/threat_intelligence_cypress/config.ts
   - x-pack/test/functional_enterprise_search/visual_config.ts
   - x-pack/test/functional_enterprise_search/cli_config.ts
+  - x-pack/test_serverless/functional/test_suites/security/cypress/config_headless.ts
+  - x-pack/test_serverless/functional/test_suites/security/cypress/config_visual.ts
```

Comment:
This will cause both suites to run in CI.  Is there any coverage difference?

Reply:
@jbudz this was added here due to a linter error. These 2 configurations are for executing Cypress on the local machine on headless and visual mode. If everybody agrees on the initial sctructure another command should be created to execute cypress on the CI.